9_ninth_hw/1_first_task.py

```diff
@@ -0,0 +1,138 @@
+"""
+Create a class describing any company. For example, Toshiba or Apple
+
+I will evaluate the completeness of the described class and will withdraw points for everything unnecessary.
+I want to see clean code. I expect to see clean code with annotations.
+So far, no first-class connections with the second. In all methods, I expect to see docstrings with a sane description.
+"""
+from datetime import date, timedelta, datetime
+
+
+class Apple:
+    BRAND = "Apple"
+
+    def __init__(self, __website: str, __address: str, __employees: int):
+        self.__website = __website
+        self.__address = __address
+        self.__employees = __employees
+
+    @property
+    def employees(self):
+        return self.__employees
+
+    @employees.setter
+    def employees(self, new_employees_amount: int):
+        self.__employees = new_employees_amount
+
+    def new_employee(self, new_employees_num: int):
+        """
+        :param new_employees_num: update number of employees by the new employees
+        :return: new number of employees
+        """
+        self.__employees += new_employees_num
+        return self.__employees
+
+    def fire_employee(self, fire_employees_num: int):
+        """
+        :param fire_employees_num: update number of employees by reduction of employees amount
+        :return: new number of employees
+        """
+        self.__employees -= fire_employees_num
+        return self.__employees
+
+    @property
+    def website(self):
+        """
+        :return: website name
+        """
+        return self.__website
+
+    @website.setter
+    def website(self, new_website_page: str):
+        """
+        :param new_website_page: new website page value
+        :return: a new website page
+        """
+        self.__website = new_website_page
```

Comment:
let's add some validation here. Like new value matches some pattern 
for example www.[A-z].(com|ua)